browser/pages/account/controller.js

```diff
@@ -62,6 +63,18 @@ class AccountController {
     this.tandcNotSigned = false;
   }
 
+  save(event) {
+    let checkbox = document.getElementById('rememberMe');
+    localStorage.setItem('rememberMe', checkbox.checked);
+    if (event.target.checked === false) {
```

Comment:
@sudhirverma You can directly use `checkbox` variable here instead of calling the `$event` from html view.

```javascript
  save() {
    let checkbox = document.getElementById('rememberMe');
    localStorage.setItem('rememberMe', checkbox.checked);
    if (!checkbox.checked) {
      let dataFilePath = path.join(Platform.localAppData(), 'settings.json');
      if(fs.existsSync(dataFilePath)) {
        TokenStore.deleteItem('login', this.installerDataSvc.username);
        rimraf.sync(dataFilePath);
      }
    }
  }
```
and in the html remove the `$event` call from the function.

```html
<input class="remember-me" id="rememberMe" type="checkbox" ng-model = "acctCtrl.rememberMe" ng-click="acctCtrl.save()"><span class="remember-me-label">Remember Me</span>
```

Reply:
thanks @mohitsuman :)